wire-ios-request-strategy/Sources/Request Strategies/MLS/Actions/FetchMLSConversationGroupInfoActionHandler.swift

```diff
@@ -37,7 +37,7 @@ final class FetchMLSConversationGroupInfoActionHandler: BaseFetchMLSGroupInfoAct
             for: action,
             path: "/conversations/\(action.domain)/\(action.conversationId.transportString())/groupinfo",
             apiVersion: apiVersion,
-            minRequiredAPIVersion: .v3
```

Comment:
Are you sure this is only available from `v4` ? IIRC we were already fetching the group info while we were on `v3`

Reply:
I don't see this listed on API V3 Swagger for Anta.

Reply:
Ok, as long as it works 🤷 I'm curious why it doesn't show up on swagger though

Reply:
Maybe it moved from v3 to v4?